Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/AddResolverController.java

```diff
@@ -50,4 +58,26 @@ public String addResolver(Principal principal,
 
         return "redirect:/auth/resolvers";
     }
+
+    @RequestMapping(method = RequestMethod.POST, value = "auth/resolvers/testAdd")
+    public @ResponseBody ResponseEntity<String> testAddPageSuccess(@RequestParam("data") JSONObject data)
+            throws JSONException {
+
+        IProjectHandleResolver projectHandleResolver = new ProjectHandleResolver();
+
+        projectHandleResolver.setProjectName(data.getString("projectName"));
+        projectHandleResolver.setDescription(data.getString("projectDescription"));
+        projectHandleResolver.setProjectUrl(data.getString("projectUrl"));
+        projectHandleResolver.setHandlePattern(data.getString("handlePattern"));
+        projectHandleResolver.setResolvedHandlePattern(data.getString("resolvedHandlePattern"));
+        projectHandleResolver.setHandleExample(data.getString("handleExample"));
+        projectHandleResolver.setResolvedHandleExample(data.getString("resolvedHandleExample"));
```

Comment:
still, why are you building a complete resolver here? You don't need anything besides the handles and patterns, right?